api/controller/migrationmaster/client.go

```diff
@@ -142,12 +143,23 @@ func (c *Client) ModelInfo() (migration.ModelInfo, error) {
 	if err != nil {
 		return migration.ModelInfo{}, errors.Trace(err)
 	}
+
+	var modelDescription description.Model
+	if bytes := info.ModelDescription; len(bytes) > 0 {
+		var err error
+		modelDescription, err = description.Deserialize(info.ModelDescription)
+		if err != nil {
+			return migration.ModelInfo{}, errors.Trace(err)
+		}
+	}
+
 	return migration.ModelInfo{
 		UUID:                   info.UUID,
 		Name:                   info.Name,
 		Owner:                  owner,
 		AgentVersion:           info.AgentVersion,
 		ControllerAgentVersion: info.ControllerAgentVersion,
+		ModelDescription:       modelDescription,
```

Comment:
Could you please update at least one of the tests (prob `TestModelInfo()`) so it has something in `ModelDescription`?

Reply:
Sure.